### PR TITLE
Described a first look at the report designer

### PR DESCRIPTION
**Pull request by MarySammal**

### CODE REVIEW

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/first-look-at-the-report-designer.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+title: First Look at the Report Designer
+owner: Mary Sammal
+---
+# First Look at the Report Designer
+
+![](../../../images/eurd-win-report-designer.png)
+
+The report _design surface_ displays a report's structure and contents. The Report Designer's tools provide capabilities to design the report:
+
+- observe the report's data source schema in the [Field List](report-designer-tools\ui-panels\field-list.md);
```
[the remark this answers is not in the document]
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/first-look-at-the-report-designer.md]: I would change it to "Access the report's data source schema..." 
Ok?

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/first-look-at-the-report-designer.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+title: First Look at the Report Designer
+owner: Mary Sammal
+---
+# First Look at the Report Designer
+
+![](../../../images/eurd-win-report-designer.png)
+
+The report _design surface_ displays a report's structure and contents. The Report Designer's tools provide capabilities to design the report:
```
[the remark this answers is not in the document]
Comment (MarySammal) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/first-look-at-the-report-designer.md]: Can I change to the following:

The report _design surface_ displays a report's structure and contents. The Report Designer's panels provide tools to design the report:

[the remark this answers is not in the document]
Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/first-look-at-the-report-designer.md]: You can use the tools on the Report Designer's panels to design the report:

File: interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/first-look-at-the-report-designer.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dirk-pieterse) [reply interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/first-look-at-the-report-designer.md]: Ok

**Pull request merged**